cli/internal/cmd/iamupgradeapply.go

```diff
@@ -6,11 +6,14 @@ SPDX-License-Identifier: AGPL-3.0-only
 package cmd
 
 import (
+	"context"
```

Comment:
Can you test that IAM upgrades still works (e.g. remove one permission and check if diff is applied)? Writing a test still seems different in the current code stage but would ofc be more sustainable

Reply:
Manual tests ran without problem.
I will start a couple automated upgrade tests later.